mta-v7.x/README.md

```diff
@@ -12,6 +12,15 @@ the cloud-readiness compatibility of their code.
 - `repositoryUrl` [mandatory] - the git repo url to examine
 - `recipients` [mandatory] - A list of recipients for the notification in the format of `user:<namespace>/<username>` or `group:<namespace>/<groupname>`, i.e. `user:default/jsmith`.
 
+# Requirement for Repository
+- Create a git repo with a simple Java class. It can either be public or private.
+  Currently, MTA supports mostly Java projects.
+- If repository is private, refer to [configuring repository][4] on how to set up credentials on MTA.
+    - Note: When creating the source control credentials in MTA, ensure to input personal access token in the password field.
+      Refer to [personal access token] [5] on how to create personal access token (classic) on GitHub.
+    - Also, git url address must use https. Refer to [about remote repositories][6] for more details.
+- If the repo is public, no further configuration is needed.
+
```

Comment:
Instead of copy & paste this paragraph in multiple README, can we consider adding the paragraph [here](https://github.com/parodos-dev/serverless-workflows/tree/main/mta#readme) and then only add a link in each `mta-v*` README? Or maybe it's not a good place?